variants/vmware-k8s-1.28/Cargo.toml

```diff
@@ -32,6 +32,7 @@ kernel-parameters = [
 ]
 included-packages = [
     # core
+    "early-boot-config-vmware",
```

Comment:
@bcressey Your potential solution might change this, but is this the correct approach for now?

Reply:
> @bcressey Your potential solution might change this, but is this the correct approach for now?

Yes, this is the best way for now.